pages/careers/_scripts.html

```diff
@@ -0,0 +1,32 @@
+<script>
+  $(document).ready(function() {
+
+    // Scroll the page to the selected element
+    // Copied from https://stackoverflow.com/a/4801719/2308858
+    (function($) {
+      $.fn.scrollTo = function() {
+        $('html, body').animate({
+          scrollTop: $(this).offset().top + 'px'
+        }, 'fast');
+        return this; // for chaining...
+      }
+    })(jQuery);
+
+    // When the user clicks a reference, scroll to the corresponding citation.
+    (function($) {
+      $.fn.addScrollForCitation = function(num) {
+        $("#ref-" + num ).click(function() {
+          $("#citation-" + num).scrollTo();
+        });
+        return this; // for chaining...
+      }
+    })(jQuery);
+
+    $().addScrollForCitation(1);
+    $().addScrollForCitation(2);
+    $().addScrollForCitation(3);
+    $().addScrollForCitation(4);
+    $().addScrollForCitation(5);
+
+  });
+</script>
```

Comment:
We already have scrolling functionality: https://github.com/gruntwork-io/gruntwork-io.github.io/blob/master/assets/js/main.js#L36. All you have to add is a link to an anchor tag (`<a href="#foo">...`) and it'll add smooth scrolling automatically. I'd recommend removing the new scripts you added.

Reply:
Per above, reverted.